### PR TITLE
feat(dnsmasq): add dnsmasq to the setup scripts

### PR DESCRIPTION
**Pull request by gq0**

### CODE REVIEW

File: packer/scripts/install-dnsmasq.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -eu -o pipefail
+
+echo "Installing dnsmasq..."
+sudo apt-get update update -y
```
Comment (gq0) [inline packer/scripts/install-dnsmasq.sh]: weird double update :thinking: but the other script did this as well

Comment (jhedev) [reply packer/scripts/install-dnsmasq.sh]: Let's just remove it

**Review by jhedev — CHANGES_REQUESTED** [review]

**Comment by gq0** [comment]
all right, removed it @jhedev

**Comment by jhedev** [comment]
@gq0 FYI: I'm testing this right now

**Comment by gq0** [comment]
Awesome :+1:

**Review by jhedev — APPROVED** [review]

**Pull request merged**